moto/ec2/responses/spot_fleets.py

```diff
@@ -89,6 +91,16 @@ def request_spot_fleet(self) -> str:
         <item>
             <spotFleetRequestId>{{ request.id }}</spotFleetRequestId>
             <spotFleetRequestState>{{ request.state }}</spotFleetRequestState>
+            {% for resource_type in request.tags %}
```

Comment:
I don't think this is correct, according to the model.  I think you need to drop this line and modify your next `for` loop (see my comment there) to only get `spot-fleet-request` tags.